cpp/src/decisiontree/kernels/quantile.h

```diff
@@ -17,7 +17,7 @@
 #pragma once
 #include "../memory.h"
 template <typename T, typename L>
-void preprocess_quantile(const T *data, const unsigned int *rowids,
+void preprocess_quantile(T *data, const unsigned int *rowids,
```

Comment:
Why is the 'const'ness of `data` removed?

Reply:
hmm. was a temporary work around for compiler. pushed the correct fixes.